test/e2e/handler/nnce_conditions_test.go

```diff
@@ -110,11 +141,33 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			resetDesiredStateForNodes()
 		})
 
-		It("[test_id:3795]should have Failing ConditionType set to true", func() {
-			failingEnactmentConditions := []interface{}{
+		It("[test_id:3795][parallel] should have Failing ConditionType set to true", func() {
+			for _, node := range nodes {
+				By(fmt.Sprintf("Check %s failing state is reached", node))
+				enactmentConditionsStatusEventually(node).Should(ConsistOf(failingEnactmentConditions...), "should eventually reach failing conditions at enactments")
+			}
+			By("Check policy is at degraded state")
+			waitForDegradedTestPolicy()
+
+			By("Check that the enactment stays in failing state")
+			var wg sync.WaitGroup
+			wg.Add(len(nodes))
+			for i, _ := range nodes {
+				node := nodes[i]
+				go func() {
+					defer wg.Done()
```

Comment:
we are missing the `defer GinkgoRecover`

Reply:
Added